docs/data/charts/bars/bars.md

```diff
@@ -54,3 +54,10 @@ You can use the `stackOffset` and `stackOrder` properties to define how the seri
 By default, they are stacked in the order you defined them, with positive values stacked above 0 and negative values stacked below 0.
 
 For more information, see [stacking docs](/x/react-charts/stacking/).
+
+## Pivoting
```

Comment:
I would probably not use the term "pivoting" because even with charts it can be used to describe the "pivoting" concept of the grid: https://support.microsoft.com/en-us/office/create-a-pivotchart-c1b1e057-6990-4c38-b52b-8255538e7b1c

Reply:
Agreed, I'd also urge to go with a different heading. It looks like simply going with layout would work... 🤷 
```suggestion
## Layout
```